Hangout-Bot/Code.js

```diff
@@ -0,0 +1,99 @@
+//boolean indicating if the creation phase has started.
+let creating = false;
+
+// variable to keep track of DSA creation phase.
+var phase_num = 0;
+
+//POST query string to be built by bot.
+let query_string = '';
```

Comment:
Seems like a lot of the code in this file is not actually used currently. It would be in better practice to delete the unused code and only keep what is currently functional. This way it is easier for the reader to understand what the code currently does.
You can add TODO's if you want to indicate what you intend to do, but you generally shouldn't have variables/logic that are unused.

Reply:
Sorry about that, I'll keep that in mind next time.